examples/calendar/backend/services/appointments/controller.ts

```diff
@@ -0,0 +1,36 @@
+import { NextFunction, Request, Response } from "express";
+import { Appointment } from "../../models";
+import { ethers, BigNumber } from "ethers";
+
+export const create = async (req: Request, res: Response, next: NextFunction) => {
+	try {
+		const secret = BigNumber.from(ethers.utils.randomBytes(Math.floor(Math.random() * 100)));
+		const userId = (req as any).user.payload.id;
```

Comment:
1. why are we losing typed req to any req?
2. user or payload can be undefined.

Reply:
Updated the code appropriately.

Reply:
I don't see the updated code. Can you please recheck?

Reply:
@vaibhavgeek  kindly look into this before resolving the comment